install_config/router/f5_router.adoc

```diff
@@ -35,11 +34,8 @@ HAProxy router].
 
 [IMPORTANT]
 ====
-Support relationships between F5 and Red Hat provide a full scope of support for
-F5 integration. F5 provides support for the *F5 BIG-IP®* product. Both F5 and
-Red Hat jointly support the integration with Red Hat OpenShift. While Red Hat
-helps with bug fixes and feature enhancements, all get communicated to F5
-Networks where they are managed as part of their development cycles.
+Support relationships between F5 and Red Hat provide a full scope of support for both models of 
```

Comment:
I don't think the new statement is clear enough to help customers know who to get support from. @sferich888, what do you think?

Reply:
I was attempting to show that there are two integrations today and there will be one in the future, while stating that both companies are cooperating in supporting our joint customers. 

would it be better to say that requests for support of the F5 router plug-in should be directed first to Red Hat, while requests for support of the F5 BIG-IP Controller for OpenShift should be directed to F5 first?

Reply:
I will await your thoughts on this before suggesting content for this section

Reply:
Yes, lets be explicit on who supports what. You could even refer F5 router plug-in as 'deprecated from 3.11'. 
/cc @ironcladlou Correct on the deprecation plan?

Reply:
@rajatchopra @kalexand-rh I added proposed support language to line 38. Does this seem reasonable to you?

Reply:
/lgtm